Tribler/Core/Modules/MetadataStore/OrmBindings/torrent_metadata.py

```diff
@@ -61,4 +62,56 @@ def get_auto_complete_terms(cls, keyword, max_terms, limit=100):
                     all_terms.add(term)
             return list(all_terms)
 
+        @classmethod
+        @db_session
+        def get_random_torrents(cls, limit):
+            """
+            Return some random torrents from the database.
+            """
+            return TorrentMetadata.select().random(limit)
```

Comment:
Is it intended that there is not filtering on `REGULAR_TORRENT` here?

Reply:
The filtering should be applied, indeed. Otherwise, the thing will return both objects of class `TorrentMetadata` and `ChannelMetadata`, that is inherited from it.
This really depends on what we are trying to achieve with this. I would add an option like `regular_torrent_only=True` by default.

Reply:
Thanks! Forgot about that.